packages/orbit-components/src/SmartPassIllustrations/SmartPassIllustrations.stories.tsx

```diff
@@ -10,63 +14,51 @@ const SIZE_OPTIONS = {
   DISPLAY: "display",
 };
 
-export const Default = () => {
-  return <SmartPassIllustrations.SmartPassV1 />;
+type SmartPassPropsAndCustomArgs = React.ComponentProps<typeof SmartPassV1> & {
```

Comment:
I'm thinking if it wouldn't be (maybe) better to replace `React.ComponentProps<typeof SmartPassV1>` by general `Props` imported from `./types`:
```
type SmartPassPropsAndCustomArgs = Props & {  illustration: string; };
```

The functionality is still the same, in this story, we're switching between 5 similar components (`SmartPassV1,...SmartPassV5`), it's just a visual thing within the file to make the code a little bit simpler (?).

Point to the discussion. :D

Reply:
Good idea.

I also removed `component: SmartPassV1,` as per docs, _The default export must contain a title property that can be read statically or a component property from which an automatic title can be computed._, so I believe in this case the title property is enough and we don't have to then import `SmartPassV1` at all.